src/index.ts

```diff
@@ -57,30 +57,52 @@ async function _runMain(command: CommandType) {
         await _runPostJob(command);
     } else if (command == CommandType.All || command == CommandType.Run) {
         // Run main
-        if (_featureIsEnabled(Features.Scanning)) {
-            await _getExecutor(MicrosoftSecurityDevOps).runMain();
-        } else {
+        // If container-mapping is the only enabled tool, then skip scanning
+        if (_toolIsEnabledOnInput(Inputs.Tools, Tools.ContainerMapping, true)) {
             console.log("Scanning is not enabled. Skipping...");
+        } else {
+            await _getExecutor(MicrosoftSecurityDevOps).runMain();
         }
     } else {
         throw new Error(`Invalid command type for the main task: ${command}`);
     }
 }
 
 /**
- * Returns true if the feature is enabled in the inputs.
- * @param featureName - The name of the feature. 
- * @returns True if the feature is enabled in the inputs.
+ * Returns true if the tool is enabled on either the tools or includeTools inputs.
+ * @param toolName - The name of the tool. 
+ * @returns True if the tool is enabled in the inputs.
+ */
+function _toolIsEnabled(toolName: string): boolean {
+    let enabled: boolean = false;
+
+    enabled = _toolIsEnabledOnInput(Inputs.Tools, toolName, false);

```

Comment:
NIT: shouldn't this be `isOnlyTools: true`?

```suggestion
    enabled = _toolIsEnabledOnInput(Inputs.Tools, toolName, true);
```